ts/store/reducers/entities/messages/transformers.ts

```diff
@@ -18,10 +19,13 @@ import {
  */
 export const toUIMessage = (messageFromApi: PublicMessage): UIMessage => {
   const enriched = messageFromApi as EnrichedMessage;
+  // TODO: replace with actual types from API, for now use it if available
+  const category: MessageCategory | undefined = (messageFromApi as any)
```

Comment:
this PR LGTM

Just one point:
Could we leave a JIRA dedicated ticket in the TODO? `any` in the code base are often source of unexpected behaviors/bug. We should we sure it is fixed when the feature will be shipped

Reply:
Good point. Added in https://github.com/pagopa/io-app/pull/3546/commits/5c8f54e2d33eefc9ec32ab6d70dd49c1e54add9b